tests/Api/Request/ParametersBuilderTest.php

```diff
@@ -138,7 +140,7 @@ public function providerGetCategoryName()
                     2 => ['parentCategoryId' => 1, 'name' => 'Test1'],
                     1 => ['parentCategoryId' => 0, 'name' => 'Test0']
                 ],
-                'Test0_Test1'
+                ''
```

Comment:
There is actually an issue with the test itselft which produces an unexpected exception:
* Delete `providerGetCategoryName`.
* Replace `testGetCategoryName` with
```
    public function testGetCategoryName()
    {
        $parentCategoryMock = $this->getMockBuilder(Category::class)->disableOriginalConstructor()->getMock();
        $parentCategoryMock->parentCategoryId = 0;
        $details = new \stdClass();
        $details->name = 'Test0';
        $parentCategoryMock->details = [$details];

        $categoryMock = $this->getMockBuilder(Category::class)->disableOriginalConstructor()->getMock();
        $categoryMock->parentCategoryId = 1;
        $details = new \stdClass();
        $details->name = 'Test1';
        $categoryMock->details = [$details];

        $this->categoryService->expects($this->once())->method('get')->willReturn($parentCategoryMock);

        $parametersBuilderMock = $this->getParametersBuilderMock();

        $this->assertEquals('Test0_Test1', $parametersBuilderMock->getCategoryName($categoryMock));
    }
```